plugins/ssl/certificate_file_expiry

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+: << =cut
+=head1 NAME
+
+certficate_file_expiry - check the certificate validity of your certfificates
+
+= head1 CONFIGURATION
+
+Installing: Add list of your certificates prefixed by the type in munin plugin-conf.d
+
+For openvpn ca.crt and crl.pem
+[cert_validity]
```

Comment:
A remainder of the old name is hiding here :)

Reply:
thanks for noticing, fixed